pkg/chains/spire/spire_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2021 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spire
+
+import (
+	"context"
+	"testing"
+
+	logtesting "knative.dev/pkg/logging/testing"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func TestVerify(t *testing.T) {
+	spireWorkloadAPI := NewSpireWorkloadApiClient("")
```

Comment:
may need to come up with some interface abstraction here to have a test client which can use some custom verification code.